src/Helper/Commands.ts

```diff
@@ -42,8 +42,8 @@ export class Commands {
      * @param publisherPrefix Publisher Prefix
      * @param publisherName Publisher Name
      */
-    public static Force(namespace: string, controlname: string, template: string, pkg: number, publisherPrefix: string, publisherName: string) {
-        return `yo pcf --ns ${namespace} --n ${controlname} --t ${template} --pkg ${pkg} --pp ${publisherPrefix} --pn ${publisherName} --force`;
+    public static Force(namespace: string, controlname: string, template: string, pkg: number, publisherPrefix: string, publisherName: string, solutionType: string) {
+        return `yo pcf --ns ${namespace} --n ${controlname} --t ${template} --pkg ${pkg} --pp ${publisherPrefix} --pn ${publisherName} --spt ${solutionType} --force`;
```

Comment:
Need to make sure the package.json file is refering to the minimum version of your PCF Generator. If someone is using the old version of PCF Generator will get errors when using this PCF Builder version. Bumping up the version in pacakge.json file will make sure latest version will be downloaded as part of dependency.

Reply:
Forgot about the package json part, I'll bump the version to the latest one.